.swiftlint.yml

```diff
@@ -1 +1 @@
-disabled_rules:
- trailing_whitespace
opt_in_rules:
- empty_count
- empty_string
excluded:
- Carthage
- Pods
- DerivedData/FantasyUniverse/SourcePackages

line_length:
    warning: 150
    error: 200
    ignores_function_declarations: true
    ignores_comments: true
    ignores_urls: true
function_body_length:
    warning: 300
    error: 500
function_parameter_count:
    warning: 6
    error: 8
type_body_length:
    warning: 300
    error: 500
file_length:
    warning: 1000
    error: 1500
    ignore_comment_only_lines: true
cyclomatic_complexity:
    warning: 15
    error: 25
reporter: "xcode"
\ No newline at end of file
+disabled_rules:
- trailing_whitespace
opt_in_rules:
- empty_count
- empty_string
excluded:
- Carthage
- Pods
- DerivedData/FantasyUniverse/SourcePackages

line_length:
    warning: 200
    error: 250
    ignores_function_declarations: true
    ignores_comments: true
    ignores_urls: true
function_body_length:
    warning: 300
    error: 500
function_parameter_count:
    warning: 6
    error: 8
type_body_length:
    warning: 300
    error: 500
file_length:
    warning: 1000
    error: 1500
    ignore_comment_only_lines: true
cyclomatic_complexity:
    warning: 15
    error: 25
reporter: "xcode"

```

Comment:
Are you editing this in TextEdit? Please don't, it messes up the end end of line sign as you can see. Also, may be interesting for you to challange yourself with 120 line lenghts, like we have in the project 😅

Reply:
Ok, use Sublime Text e.g. instead ?